dist/index.js

```diff
@@ -27206,41 +27206,46 @@ utils.intFromLE = intFromLE;
 
 },{"bn.js":19,"minimalistic-assert":77,"minimalistic-crypto-utils":78}],48:[function(require,module,exports){
 module.exports={
-  "_from": "elliptic@^6.2.3",
+  "_args": [
+    [
+      "elliptic@6.4.1",
+      "/Users/gavinatkinson/Tlon/urbit-key-generation"
```

Comment:
I was gonna ignore `dist` contents, but these local paths don't seem quite right...

Reply:
I think we should gitignore /dist. Can do so in another PR.

Reply:
I didn't realize generate_test_wallets.js did basically the same thing. I can just remove the /tools dir in that case.

Reply:
Agreed about gitignore!

Reply:
I have gitignored dist